package.json

```diff
@@ -24,7 +24,8 @@
     "build": "webpack --config webpack.config.js; cp example/index.html dist/",
     "lint": "eslint --ext .js,.jsx .",
     "test": "cross-env NODE_ENV=test mocha --reporter progress --compilers js:babel-core/register --require ignore-styles --recursive \"./components/**/*.spec.js\"",
-    "test:watch": "npm run test -- --watch"
+    "test:watch": "npm run test -- --watch",
+    "build-navbar": "webpack --config navbar.webpack.config.coffee; cp index.html dist/"
```

Comment:
We can add `--bail --progress --build --tc` flags to the `build-navbar` command. @aselbie  Can you confirm?